content/about/about-content.md

```diff
@@ -2,26 +2,46 @@
 
```

Comment:
*This is really important: this page is the place where we keep **most** of the content for the about page, but the canonical source for the about page is actually https://github.com/edgi-govdata-archiving/EEW-Website/blob/main/src/pages/about.js. The contents of this markdown file are embedded into that .js file, so if you only look at this markdown file, you're only getting a part of the story*

Reply:
*This is the line where it's embedded https://github.com/edgi-govdata-archiving/EEW-Website/blob/main/src/pages/about.js#L16*

*That line uses information it gets from this query for "content": https://github.com/edgi-govdata-archiving/EEW-Website/blob/main/src/pages/about.js#L41*

*And it knows what "content" is because we define it here: https://github.com/edgi-govdata-archiving/EEW-Website/blob/main/content/about/about.json#L3*